travis/utfcheck.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Maybe add a comment starting on line 2 explaining what this script does (look at the similar script in op-desafios, but adapt the comments to this case.)

Reply:
Done